freqtrade/rpc/telegram.py

```diff
@@ -307,11 +307,14 @@ def _balance(self, bot: Bot, update: Update) -> None:
             result = self._rpc_balance(self._config.get('fiat_display_currency', ''))
             output = ''
             for currency in result['currencies']:
-                output += "*{currency}:*\n" \
-                          "\t`Available: {available: .8f}`\n" \
-                          "\t`Balance: {balance: .8f}`\n" \
-                          "\t`Pending: {pending: .8f}`\n" \
-                          "\t`Est. BTC: {est_btc: .8f}`\n".format(**currency)
+                if currency['est_btc'] > 0.0001:
```

Comment:
This depends on the price of BTC/USD but I think we are safe for the next couple of years ;)

Reply:
yes it does - but using fiat currency is difficult as that can be configured - and 1$ is not 1rubel (however much that is ...)...